src/commands/run_qa.yml

```
@@ -81,6 +86,7 @@ steps:
               --skip-update \
               --token "${<< parameters.token >>}" \
               <<# parameters.conflict >> --conflict "<< parameters.conflict >>" <</ parameters.conflict >> \
+              --background=<<parameters.background>> \
```

Alternatively, only use the flag if `parameters.background` is `true` ?
```suggestion
              <<# parameters.background >> --background <</ parameters.background > \
```

---

There's a slight difference in semantics - the way in the original PR explicitly sets it and the suggested change relies on the implicit behavour of the CLI. I prefer explicitly setting it because then if the flag changes defaults (unlikely to occur!), it won't change the behaviour in the orb.

It's also shorter :grin:

---

I made the whitespace consistent for `<< parameters.background >>`, and added quotes.